### ISSUE TITLE
adegenet tutorials

### ISSUE THREAD

**Issue by thibautjombart** [NESCent/popgenInfo#31]
It's been mentioned briefly before, but do you guys want to put the adegenet tutorials on this website?
The Rnw are there:
https://github.com/thibautjombart/adegenet/tree/master/tutorials

They are pretty much ready to use, though you may want to:
1. translate Rnw into Rmd and handle them as html
2. add your name in the authors if you add something to the document; I'd like to stay first unless somebody becomes a larger contributor
3. ensure compatibility with next adegenet 2.0-0 (https://github.com/thibautjombart/adegenet); it can be installed by:
```r
library(devtools)
install_github("thibautjombart/adegenet")
library("adegenet")
```

**Comment by thibautjombart** [comment]
If these are not general enough, no worries, just let me know

**Comment by mgiresi** [comment]
They are pretty awesome. They might be a little long, but I can also work
on converting them AFTER May 8th.  I don't want authorship on anything. I
prefer being in the background.

**Comment by hlapp** [comment]
> 1. translate Rnw into Rmd and handle them as html

That's a possibility, but I'd rather like to engineer the rendering workflow such that it can take .Rnw as well.

**Comment by thibautjombart** [comment]
@mgiresi regading authorship: point is, beyond recognition, it is useful for people to know who to contact regarding a document's content. You take credit for the good, and the bad things in there .. ;)

**Comment by thibautjombart** [comment]
@hlapp is Rmd the preferred format, or is the platform agnostic?

**Comment by hlapp** [comment]
> @hlapp is Rmd the preferred format, or is the platform agnostic?

The platform R is, I think, agnostic. You can meanwhile write package vignettes either way, and package vignettes get rendered using R method calls. So we should be able to accomplish the same thing. The one thing I'm not sure about is whether this is still true for HTML (rather than PDF) as the rendering product (I understand that package vignettes get all rendered to PDF when building a package?).

**Comment by zkamvar** [comment]
> (I understand that package vignettes get all rendered to PDF when building a package?)

This is not true. Rmd vignettes are rendered as html by default [i.e. the magrittr vignette](http://cran.r-project.org/web/packages/magrittr/vignettes/magrittr.html). 

This question about [turning Rnw to html](http://stackoverflow.com/questions/7512897/how-to-turn-a-latex-sweave-file-rnw-into-html) shows that it is apparently non-trivial. There are [ways to convert latex to html](http://biostat.mc.vanderbilt.edu/wiki/Main/SweaveConvert#Current_Best_Approach_for_Conver), but if we want to do this, it would be better to find an engine to convert Rnw to markdown since we want pandoc to be able to handle the includes properly.

**Comment by thibautjombart** [comment]
Yes the conversion from LaTeX to html is non-trivial. A few years back
(during my PhD) I tried pretty much all the options around, and the main
problem was basically with formula, which have to be converted into
pixmaps. Which is really $a$nnoying when y$o$u have various symbols in-line
&x&, $y$, $z$...

**Comment by hlapp** [comment]
> This question about turning Rnw to html shows that it is apparently non-trivial. There are ways to convert latex to html, but if we want to do this, it would be better to find an engine to convert Rnw to markdown since we want pandoc to be able to handle the includes properly.

To be clear, what we want isn't to convert LaTeX (or TeX) to HTML, which is what almost all of the StackOverflow article is about, but to convert .Rnw to HTML. The OP starts with saying that they could well do this to obtain HTML:

```R
Sweave('temp1.rnw', driver = RweaveHTML)
```

but that they need TeX to HTML so it works on Windows. We don't need it to work on Windows (the build environment is Linux). I haven't had an opportunity yet to check in detail, but are there reasons why the above is probably _not_ going to work for us?
